cookbooks/tomography_based_plate_motions/2D_slice_with_faults_slabs_and_topo.prm

```diff
@@ -0,0 +1,168 @@
+# This cookbook demonstrates how we can set up instantaneous mantle flow models
+# using available geophysical constraints. To avoid large computational cost,
+# this model is a 2D spherical shell with imposed plate boundaries and driven
+# by mantle forces arising from observed mantle heterogeneity.
+# The generated mantle flow field at the surface can be compared with the observed
+# surface deformation and the fit can be used to constrain the physical state
+# of the mantle.
+# We define plate boundaries and cratons using WorldBuilder.
+
+include 2D_slice_with_faults_and_cratons.prm
```

Comment:
Can you add a brief description of what is done in this PRM file and how this cookbook modifies it?